### PR TITLE
docs: Add note on how to use cart item tax and discount amounts

### PR DESCRIPTION
**Pull request by cbetta**
Adds note on how to use cart item tax and discount amounts.

Additionally fixes some linting/spelling errors

### CODE REVIEW

File: .alexrc.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-  "allow": ["remain", "invalid"]
+  "allow": ["remain", "invalid", "special", "latin", "failed"]
```
Comment (cbetta) [inline .alexrc.json:2]: Allows for `Latin` and `failed` to be used in descriptions.

File: .spelling [hunk 2]
```diff
@@ -12,4 +12,6 @@ GoCardless
 3-D
 Mastercard
 Amex
-requestor
\ No newline at end of file
+requestor
+webhook
+webhooks
```
Comment (cbetta) [inline .spelling:17]: Adds some non-standard dictionary words.

File: reference/path-parameters/webhook_hook.yaml [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ---
 schema:
   type: string
+  example: "e4f7797af7cc56edb85cde7f94745a13"
```
Comment (cbetta) [inline reference/path-parameters/webhook_hook.yaml:4]: Adds a (not representative) example of an opaque key. Needed to pass lint checks

File: reference/paths/webhooks__inbound_webhook.yaml [hunk 4]
```diff
@@ -8,8 +8,7 @@ description: |-
   service definition.
 tags:
   - Webhooks
-security:
-  - None
+security: []
```
Comment (cbetta) [inline reference/paths/webhooks__inbound_webhook.yaml:11]: This is the correct way to specify no security.

File: reference/resources/statement_descriptor.yaml [hunk 5]
```diff
@@ -19,15 +19,17 @@ properties:
...
       Other validations:
+
       1. Contains only Latin characters.
       2. Contain at least one letter
       3. Does not contain any of the special characters `< > \ ' " *`
       4. Supports:
-        1. Lowercase: `a-z`
-        2. Uppercase: `A-Z`
+        1. Lower case: `a-z`
+        2. Upper case: `A-Z`
```
Comment (cbetta) [inline reference/resources/statement_descriptor.yaml:28]: Fix spelling.

File: reference/resources/statement_descriptor.yaml [hunk 6]
```diff
@@ -44,11 +46,11 @@ properties:
...
         3. Numbers: `0-9`
```
Comment (cbetta) [inline reference/resources/statement_descriptor.yaml:51]: Fix spelling.

File: reference/resources/statement_descriptor.yaml [hunk 7]
```diff
@@ -19,15 +19,17 @@ properties:
...
         4. Spaces: ` `
-        5. Special characters: `. , _ - ? + /`
+        5. Special characters: `. , _ - ? + /`.
```
Comment (cbetta) [inline reference/resources/statement_descriptor.yaml:31]: Added a full stop as every description needs to end with one.

File: reference/resources/statement_descriptor.yaml [hunk 8]
```diff
@@ -44,11 +46,11 @@ properties:
...
```
Comment (cbetta) [inline reference/resources/statement_descriptor.yaml:53]: Added a full stop as every description needs to end with one.

File: spectral/full-stop.js [hunk 9]
```diff
@@ -1,5 +1,5 @@
 module.exports = targetVal => {
-  if (!targetVal.trim().endsWith('.')) {
+  if (targetVal.trim && !targetVal.trim().endsWith('.')) {
```
Comment (cbetta) [inline spectral/full-stop.js:2]: Ignore descriptions that are not strings. Which happens right now because we have a field called `description`.

**Review by gryevns — APPROVED** [review]

**Pull request merged**